content/backmatter.tex

```diff
@@ -1249,4 +1249,41 @@ \section{Version 1.1}
 %
 \end{itemize}
 
+\chapter{Errata}\label{sec:errata}
+
+The \openshmem specification may occasionally include errata that are
+discovered after the release of new versions.
+These errata can range from typographical mistakes to more significant
+technical inaccuracies that may affect the implementation or understanding of
+the \openshmem API semantics.
+This Errata section reports these issues with the goal to maintain the
+integrity and utility of the OpenSHMEM specification.
+
+The sections below document all known errata, their corresponding corrections,
+and the affected version of the OpenSHMEM specification.
+It serves as a historical record of the changes made and assists users and
+implementers with applying the necessary corrections.
+
+\section{Version 1.5}
+
+\begin{itemize}
+  \item Clarified that \FUNC{shmem\_test\_all} and
+     \FUNC{shmem\_test\_all\_vector} routines return 1 when the test set is empty
+        (\href{https://github.com/openshmem-org/specification/pull/466}{\#466}).
```

Comment:
Please remove the links to github. These aren't perma-links and will stop working at some point. Can you include a reference to the changelog entry instead?

Reply:
done